### PR TITLE
updated link, content, and added annotation

### PR DESCRIPTION
**Pull request by shospital**
- updated link and text
- added annotation

### CODE REVIEW

**Comment by shospital** [comment]
@DaisyShi19 could you review my version and if it looks good, could you approve merge?

**Review by DaisyShi19 — COMMENTED** [review]
Hi Sunny, I reviewed the changes and they look good. 
The one thing is for the acknowledgments, can you revert back to the previous version? I think that one is more accurate. Thanks!

**Comment by shospital** [comment]
Is the acknowledgement you want from the main branch? didn't realize it's modified.

File: mask-shallow-ocean-color/R/mask-shallow-ocean-color.md [hunk 1]
```diff
@@ -219,5 +229,6 @@ within each ocean color pixel is in shallow water (&lt;30m depth).
 
 ## Acknowledgements
 
-Special thanks to Kisei Tanaka from NOAA's Pacific Islands Fisheries Science Center (PIFSC) for his contributions to this tutorial, which is adapted from the scripts he developed. Additionally, portions of this tutorial have been revised based on a previous version created by Melanie Abecassis and Thomas Oliver.
```
Comment (DaisyShi19) [inline mask-shallow-ocean-color/R/mask-shallow-ocean-color.md:222]: This one is to keep.

Comment (shospital) [reply mask-shallow-ocean-color/R/mask-shallow-ocean-color.md:222]: updated the ack and regenerated the markdown.

Comment (DaisyShi19) [reply mask-shallow-ocean-color/R/mask-shallow-ocean-color.md:222]: Thanks!

**Comment by shospital** [comment]
could you approve Merge pull request?  I want you to start approving Merging. :)

File: mask-shallow-ocean-color/R/mask-shallow-ocean-color.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Mask shallow pixels for satellite ocean color datasets
 
-> Updated November 2023 <br/>
+> Updated March 2023 <br/>
```
Comment (DaisyShi19) [inline mask-shallow-ocean-color/R/mask-shallow-ocean-color.md:3]: March 2024 perhaps :)

**Comment by DaisyShi19** [comment]
> could you approve Merge pull request? I want you to start approving Merging. :)

Yes, how do I approve?

**Comment by shospital** [comment]
in RED, you should see merge pull request. just check merge without wiating for requirements

**Review by DaisyShi19 — APPROVED** [review]

**Pull request merged**